.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "cinch"]
 	path = cinch
-	url = ../cinch.git
+	url = https://github.com/losalamos/cinch.git
```

Comment:
The reason why we use relative paths is that we are mirroring vpic onto machines, which don't have internet access.